wled00/const.h

```diff
@@ -8,10 +8,12 @@
 #define GRADIENT_PALETTE_COUNT 58
 
 //Defaults
+#ifndef USE_MY_CONFIG
 #define DEFAULT_CLIENT_SSID "Your_Network"
 #define DEFAULT_AP_SSID     "WLED-AP"
 #define DEFAULT_AP_PASS     "wled1234"
 #define DEFAULT_OTA_PASS    "wledota"
+#endif
```

Comment:
this should then also be added to the my_config_sample
this removed warnings from the compiler when replacing these from my_config

Reply:
I agree with @blazoncek - removing preprocessor #define will save exactly zero bytes by itself. But it makes the code harder to read.

Reply:
Perhaps a CUSTOM_AP_SSID, CUSTOM_AP_PASS, etc, but that would make for even clunkier code.
Wrapping this in the WLED_DISABLE_AP I proposed would also work, as these are not needed when the AP function of the WIFI is disabled. Wifi still works, it just required compiling in the wifi-password. And since this is a custom build only thing, that should work just fine.